src/test/java/com/epam/tc/hw3/pages/DifferentElementsPage.java

```diff
@@ -0,0 +1,52 @@
+package com.epam.tc.hw3.pages;
+
+import com.epam.tc.hw3.pages.components.ColorsDropDownComponent;
+import com.epam.tc.hw3.pages.components.LogComponent;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class DifferentElementsPage {
+
+    private ColorsDropDownComponent colorsDropDownComponent;
+
+    private LogComponent logComponent;
+
+    @FindBy(xpath = "//label[@class='label-checkbox'][1]/*")
+    private WebElement waterCheckBox;
+
+    @FindBy(xpath = "//label[@class='label-checkbox'][3]/*")
+    private WebElement windCheckBox;
+
+    @FindBy(xpath = "//label[@class='label-radio'][4]/*")
+    private WebElement radioSelen;
```

Comment:
It's not a good approach to use indexes in the locators. Only in special situations. And this is not that one

Reply:
done